### DISCUSSION TITLE
Suggestion : RE2 Add random enemies in Boss Room and Items in enemies body

### DISCUSSION THREAD

**Discussion by otamega72** [IntelOrca/biorand#416] [Ideas]
1)  RE2 Allow to add random enemies in Boss Room  : 303 , 309 , 504 , 617 ,  702 and (700 - for Scenario A)
2)  Allow to randomize items in enemies body. (killed enemies to grab items)

Thanks.

**Comment by IntelOrca** [comment]
There are reasons why random enemies do not / can not appear in these rooms. Any room containing a cutscene or boss complicates the process of adding random enemies. Room 504 already has random enemies. 309 has random enemies for Leon.

Adding new items is not something randomizer currently does, and it would be complicated to spawn them if enemies are killed. They would also have to be low priority only, since you can't always rely on the player being able to kill the enemy in order to collect item.
